src/main/views/courts/tabs/phoneNumbersContent.njk

```diff
@@ -107,30 +115,43 @@
     value: contact.explanation_cy
   }) }}
 
-  {{ govukButton({
-    text: "Remove",
-    name: "deletePhoneNumber",
-    type: "button",
-    classes: "govuk-button--secondary deletePhoneNumber"
-  }) }}
+  {% if isNew === false %}
+    {{ govukButton({
+      text: "Remove",
+      name: "deletePhoneNumber",
+      type: "button",
+      classes: "govuk-button--secondary deletePhoneNumber"
+    }) }}
+  {% endif %}
+
+  {% if isNew === true %}
```

Comment:
I think you can do

{% if isNew %}

or 

{% if !isNew %}

? Maybe? ignore if there is another obvious reason

Reply:
You can do the former but not the latter. I think you can use a 'not' keyword instead of an exclamation mark but I've used === throughout just to keep it consistent.